### DISCUSSION TITLE
Apple Privacy file

### DISCUSSION THREAD

**Discussion by eppisapiafsl** [TheWidlarzGroup/react-native-video#3536] [Q&A] [answered]
### What are you trying to do?

Starting April 2024, Apple will request a privacy file ([App Privacy Details - App Store - Apple Developer](https://developer.apple.com/app-store/app-privacy-details/)) to track the "data collection".

I saw the "react-native-video" references the `FBLPromises` SDK listed in the Apple SDK list that requires the App Privacy detail.

Is this something on your radar? Will there be a release (beta) with the App Privacy file, or do we not need it?

### Platforms

iOS

**Comment by KrzysztofMoch** [comment]
There is a PR https://github.com/google/promises/pull/226/files that implement it - I will bump version of this dep when it will be released

I don't think that we will need that in react-native-video, but will check that

**Comment by KrzysztofMoch** [comment, marked as answer]
We are not using promises anymore so I guess it fine now
